### DISCUSSION TITLE
I don't understand RSSI

### DISCUSSION THREAD

**Discussion by szczukot** [arendst/Tasmota#21370] [Q&A]
I have something like that
MQT: domoticz/in = {"idx":711,"nvalue":0,"svalue":"","Battery":100,"RSSI":6}
what does this RSSI mean?
This is not RSSI (in dBm) and not Quality (in percentage).

**Comment by sfromis** [comment]
If you instead look at the STATE messages, you'll see two values like: `"RSSI":100,"Signal":-40`

The `Signal` field is the usual RSSI numbers, while the history is that the field name "RSSI" was instead used for a "humanized" version, basically similar to a percentage, where 100 is "really good", and downwards from there, with 50 already being bad.

To be clear, this is not specifically about what you see in the Domoticz payload, and if further calculation was done there.

**Reply by sfromis** [reply]
Checked the Domoticz driver, and it looks like it takes the quality number and divides by 10, meaning that the value 6 is 60-69, meaning about -66 dBM value.

**Reply by szczukot** [reply]
![image](https://github.com/arendst/Tasmota/assets/10205661/248a8bf2-2327-4109-92a7-3e00bbb13fc9)
Domoticz loads it as is.

So looking at domoticz, I still don't know what "6" means

**Reply by sfromis** [reply]
You may have written while I added the additional response explaining the calculation.

**Reply by szczukot** [reply]
I see that in domoticz it is completely inconsistent.
But most often it is probably that 0 = 0% quality, 10 = 100% quality.
So, for example, -55 dBm should probably be sent as 9 (and not as 5)

**Reply by sfromis** [reply]
Yes, -55 dBm should indeed be sent to Domoticz as 9.
